src/pages/iou/request/step/IOURequestStepTaxRatePage.tsx

```diff
@@ -47,15 +35,16 @@ function IOURequestStepTaxRatePage({
     route: {
         params: {action, backTo, iouType},
     },
-    policy,
     transaction,
     report,
-    policyCategories,
-    policyTags,
-    splitDraftTransaction,
 }: IOURequestStepTaxRatePageProps) {
     const {translate} = useLocalize();
 
+    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID ?? '-1'}`);
+    const [policyCategories] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${report?.policyID ?? '-1'}`);
+    const [policyTags] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_TAGS}${report?.policyID ?? '-1'}`);
+    const [splitDraftTransaction] = useOnyx(`${ONYXKEYS.COLLECTION.SPLIT_TRANSACTION_DRAFT}${transaction?.transactionID ?? '-1'}`);
```

Comment:
We have been using `route.params.transactionID` with `withOnyx`, but here we used `transaction?.transactionID`, I am not sure are the same in `splitDraftTransaction` case? Because `transaction` come from `withFullTransactionOrNotFound` and I see it is not handled `splitDraftTransaction`. Can you please confirm?